### PR TITLE
New Item: Irresistable Star

### LINKED ISSUE #221: schyte of the ancient
Сделать предмет сильнее, увеличив процент чистого урона с 30% до 40%, убрать из рецепта Demon's Paw и уменьшить перезарядку активной способности до 35 секунд, либо поднять на ней урон

### PR DESCRIPTION
**Pull request by shimaosu**
On Murzik`s opinion, Resolves #221

### CODE REVIEW

File: localization/english/items.yml [hunk 1]
```diff
@@ -1579,3 +1579,13 @@ DOTA_Tooltip_ability_item_golden_arrow_Note0: >-
 DOTA_Tooltip_ability_item_golden_arrow_bonus_damage: +$damage
 DOTA_Tooltip_ability_item_golden_arrow_bonus_speed: +$move_speed
 DOTA_Tooltip_modifier_item_golden_arrow_counter: Golden Arrow Multiplier
+
+DOTA_Tooltip_ability_item_irresistable_star: Irresistable Star
+DOTA_Tooltip_ability_item_irresistable_star_Description: >-
+  <h1>Пассивная: Shine</h1> All your abilities deal %magic_damage_to_pure_pct%%% pure damage.
```
Comment (ark120202) [inline localization/english/items.yml]: Use the same description as Scythe of the Ancients uses

File: game/scripts/npc/npc_items_custom.txt [hunk 2]
```diff
@@ -8627,7 +8627,69 @@
...
+			}
+			"03"
+			{
+				"var_type"				"FIELD_INTEGER"
+				"cast_range_bonus"		"300"
+			}
+			"04"
+			{
+				"var_type"				"FIELD_INTEGER"
+				"bonus_intellect"	"45"
```
Comment (ark120202) [inline game/scripts/npc/npc_items_custom.txt]: Passive bonuses should go first

Comment (ark120202) [reply game/scripts/npc/npc_items_custom.txt]: Special values should be aligned

**Review by ark120202 — CHANGES_REQUESTED** [review]

**Comment by shimaosu** [comment]
Will be reworked without quasar changes soon.

**Comment by shimaosu** [comment]
Revert hero selection changes

**Pull request closed without merging**